### PR TITLE
[GH-1993] Change version text in title screen based of player settings version

### LINKED ISSUE #1993: Show real version on top right of title scree
Now this just shows `VERSION: PRE-ALPHA`

Screenshot_20240725_171403_Champions of Mirra

### PR DESCRIPTION
**Pull request by tkz00**
Closes #1993 

## Motivation

First iteration to show the version of the application, to keep track of the changes that are available.

## Summary of changes

Retrieve the version registered on the player settings with `Application.version` and show it on top right of title screen.

## How has this been tested?

Enter to title screen, check the version is the same one you can find in the `Version` field in player settings of the unity editor, then go past the title screen to the main screen to check nothing breaks.

Reference (that's just a testing value):

Screenshot_20240725_173116_client

## Checklist
- [X] I have tested the changes locally.
- [X] I have tested the whole game after applying the changes, not only the affected areas.
- [X] I self-reviewed the changes on GitHub, line by line.
- [X] I have tested the changes in another devices.
  - [ ] Tested in iOS.
  - [X] Tested in Android.
- [ ] This change requires new documentation.
  - [ ] Documentation has been added/updated.

### CODE REVIEW

File: client/Assets/Scripts/UI/TitleScreenController.cs [hunk 1]
```diff
@@ -38,6 +43,10 @@ public class TitleScreenController : MonoBehaviour
 
     void Start()
     {
+        StringBuilder sb = new StringBuilder();
```
Comment (AngieDutra) [inline client/Assets/Scripts/UI/TitleScreenController.cs:46]: Is StringBuilder the best approach or an overkill in this particular text? I see the potential for other dynamic texts we have but I don't know if there is a performance issue here for us to use this. Or if we are suppose to use it always. What do you think? Was this a conversation with our TL?

**Review by AngieDutra — APPROVED** [review]
I tested this and worked as expected

File: client/Assets/Scripts/UI/TitleScreenController.cs (hunk 1 above)
Comment (saugon) [reply client/Assets/Scripts/UI/TitleScreenController.cs:46]: In this situation is an overkill. _StringBuilder is designed to optimize performance when performing multiple string manipulations, especially in loops or when concatenating many strings_.

Let's change it to use the + operator 👍🏻

**Review by saugon — APPROVED** [review]
Tested on iOS 👌🏻

**Review by manucamejo — APPROVED** [review]

**Pull request merged**